### ISSUE TITLE
Need a simulator version of the framework

### ISSUE THREAD

**Issue by josh1093** [gotenna/PublicSDK#22]
The current version of the SDK doesn't include binaries for the iOS simulator.  Or I don't think it does anyway ... "file" seems to agree

JOSH-MAC-120:GoTennaSDK.framework avenza$ file GoTennaSDK 
GoTennaSDK: Mach-O universal binary with 2 architectures: [arm_v7: Mach-O dynamically linked shared library arm_v7] [arm64]
GoTennaSDK (for architecture armv7):	Mach-O dynamically linked shared library arm_v7
GoTennaSDK (for architecture arm64):	Mach-O 64-bit dynamically linked shared library arm64

If I add GoTennaSDK.framework to my app I can't run on the simulator any more.

Previous versions included i386 binaries. I had to strip them with a post build step for my app before I produced a build to Apple (they won't accept apps with i386 binaries in the store).  I was fine with that solution.

**Comment by imryan** [comment]
You cannot use the SDK in the simulator as it relies on iOS's Bluetooth framework in order to function. Check out the README in the root directory for any other information.

**Comment by josh1093** [comment]
Fair enough, but my app does a bunch of other things besides connect to GoTennas.  If I include this SDK in it that means that I cannot debug using the simulator.

**Comment by josh1093** [comment]
#2 seems like this has been reported and solved before.

**Comment by imryan** [comment]
@josh1093 It was resolved on an older build of the SDK. We're going to look into building the current version to support the i386 (simulator) architecture.

**Comment by josh1093** [comment]
Any word on this?

**Comment by imryan** [comment]
@josh1093 Will take a look in the current pending iOS 11-related release and keep you updated.

**Comment by imryan** [comment]
@josh1093 The latest release supports iOS 11 (1724fb21a1a547942464116a248eecce1b83f200) but does not have `i386` architecture support just yet.
